tests/test_unit/test_cdf_tk/test_utils/test_fileio.py

```diff
@@ -322,6 +323,62 @@ def part_number(filepath: Path) -> int:
 
 
 class TestCSVReader:
+    CSV_CONTENT = """text,integer,nested,boolean,float
```

Comment:
Can you add some edge cases here such as:
- Empty file
- File cointaining only header
- File with duplicate names in the column header

Reply:
I added that and a few more cases as through usage of this (in upcoming PRs) it is very natural to start with calling this method. Then, it is good that it handles file not exists and so on.